Sources/FacebookCoreTests/GraphRequestMetadataTests.swift

```diff
@@ -21,15 +21,15 @@ import XCTest
 
 class GraphRequestMetadataTests: XCTestCase {
   func testCreatingRequestMetadata() {
-    let request = GraphRequest(graphPath: name)
+    let request = GraphRequest(graphPath: GraphPath(stringLiteral: name))
```

Comment:
Can simplify.

```suggestion
    let request = GraphRequest(graphPath: name)
```

Reply:
That's what I thought too. (actually thought that was the purpose of using ExpressibleByStringLiteral in the first place) but ran into issues when I tried to use it that way... this is probably something very simple I'm missing here.